recipes/ahead/setup.py

```diff
@@ -0,0 +1,159 @@
+import platform
```

Comment:
Can you comment on why this file is needed? Is https://github.com/Techtonique/ahead_python/blob/v0.10.0/setup.py different from this one?

Reply:
The only line in diffs is `import platform`? Does it matter much? In any case, this line is not harmful.

Reply:
It mattes b/c maintaining your own setup.py is easy to get things out of sync and hard to follow the diffs. If you really need that, can you add it as a patch  with a comment on top of it instead? That way we all know the diff and the reason for the patch.

Reply:
Fails on windows, but windows is not required in the recipe